src/platforms/hosted/platform.c

```diff
@@ -442,14 +450,15 @@ void platform_max_frequency_set(uint32_t freq)
 #endif
 
 	default:
-		DEBUG_WARN("Setting max SWD/JTAG frequency not yet implemented\n");
+		DEBUG_WARN("Setting max %s frequency not available or not yet implemented\n",
```

Comment:
Perhaps "debug interface" here instead of "SWD/JTAG"? for more consistent nomenclature through out..

This works though given the rest of the code below. We much appreciate this getting rid of some more use of "SWJ" which goes totally unexplained in the rest of the code base otherwise.